dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/domain/SpanSerializer.kt

```diff
@@ -56,9 +56,6 @@ internal class SpanSerializer(
         model.metrics.forEach {
             metricsObject.addProperty(it.key, it.value)
         }
-        // For now disable the sampling on server
-        metricsObject.addProperty(TAG_METRICS_SAMPLING_PRIORITY, 1)
```

Comment:
Shouldn't we still add this on the top level span ?

Reply:
Nope...like I explained in the PR this is already done by the "dd-trace-ot" API. They only add this attribute in the metrics if the span is the root span (a.k.a. parentId =0) I did all the tests and checked also in their code everything is fine.